float64 pos_x
float64 pos_y
bool true_victim
